3rdparty/wolfssl/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
why are there always empty lines at the top of your files?

Reply:
auto padding maybe? :*